### PR TITLE
Yuqiang/print new api dashboard

### PR DESCRIPTION
**Pull request by smallst**
fix ENG-1

image
image
image

### CODE REVIEW

File: app/views/library/dashboard/MainView.vue [hunk 1]
```diff
@@ -120,21 +151,32 @@ export default {
...
     }
 
     padding: 2rem;
+
+  }
+
+  &__download {
+    display: inline-block;
+    position: absolute;
+    top: 10px;
```
Comment (mrfinch) [inline app/views/library/dashboard/MainView.vue:178]: should use 1rem and 1.5rem to make things consistent

File: app/views/library/dashboard/components/SidebarComponent.vue [hunk 2]
```diff
@@ -68,6 +68,27 @@ export default {
 <style scoped lang="scss">
 @import "../css-mixins/variables";
 
+@media print {
+  .sidebar {
+    width: 1024px !important;
```
Comment (mrfinch) [inline app/views/library/dashboard/components/SidebarComponent.vue:73]: do we need this fixed value?

File: app/views/library/dashboard/components/helpers/FilterComponent.vue [hunk 3]
```diff
@@ -103,4 +122,16 @@ export default {
     margin-right: 5px;
   }
 }
+
+@media print {
+  .filter {
+    width: 1024px !important;
```
Comment (mrfinch) [inline app/views/library/dashboard/components/helpers/FilterComponent.vue:128]: same

**Review by mrfinch — APPROVED** [review]

File: app/views/library/dashboard/components/SidebarComponent.vue (hunk 2 above)
Comment (smallst) [reply app/views/library/dashboard/components/SidebarComponent.vue:73]: yes. this value is good for a4 page pdf. if not, on different screen the pdf may shows different

**Pull request merged**